bin/generate-globs.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * Creates a list of files to load.
+ * This creates a php file that returns an array folder names.
+ *
+ * @since 3.8
+ *
+ * @param string $base_path   Absolute path to the folder containing the files to load.
+ * @param string $type        Name to use in the messages.
+ * @param string $plugin_name Name of the plugin.
+ * @return void
+ */
+function create_load_data_file( string $base_path, string $type, string $plugin_name ): void {
+	$file_paths = glob( "{$base_path}/*/load.php", GLOB_NOSORT );
+
+	if ( ! is_array( $file_paths ) ) {
+		echo "\e[91mError while retrieving the {$type} files in {$plugin_name}\n";
+		return;
+	}
+	if ( empty( $file_paths ) ) {
+		echo "\e[91mCould not find {$type} files in {$plugin_name}\n";
+	}
+
+	sort( $file_paths, SORT_STRING | SORT_FLAG_CASE );
+
+	$write = "<?php
+/**
+ * @package {$plugin_name}
+ *
```

Comment:
Not sure this is the way to use `@package`, even more for generated file during build.
See https://docs.phpdoc.org/guide/references/phpdoc/tags/package.html

Reply:
I would be OK if modules and integrations we using their own `@package` but they don't. To me it makes more sense to use the same tag as modules and integrations.

Reply:
PHPCS enforces it.
```
Missing @package tag in file comment (Squiz.Commenting.FileComment.MissingPackageTag)
```

Reply:
Agree but then it should be part of a large "Review `@package` tags" issue, shouldn't it?